Platform/QemuBoardPkg/BoardConfig.py

```diff
@@ -35,6 +35,18 @@ def __init__(self, *args, **kwargs):
         self.BOARD_PKG_NAME       = 'QemuBoardPkg'
         self.SILICON_PKG_NAME     = 'QemuSocPkg'
 
+        #
+        # By default,
+        # _PCI_ENUM_DOWNGRADE_IO32  : 1
+        # _PCI_ENUM_DOWNGRADE_MEM64 : 1
+        # _PCI_ENUM_DOWNGRADE_PMEM64: 1
+        # _PCI_ENUM_BUS_SCAN_TYPE   : 0 (0: list, 1: range)
+        #
+        #self._PCI_ENUM_DOWNGRADE_IO32   = 0
+        #self._PCI_ENUM_DOWNGRADE_MEM64  = 0
+        #self._PCI_ENUM_DOWNGRADE_PMEM64 = 0
+        #self._PCI_ENUM_BUS_SCAN_TYPE    = 1
+        self._PCI_ENUM_BUS_SCAN_ITEMS = '0,10,0x35,128,255,0xff'

```

Comment:
what is the 255, 0xff ?   Are they same ?

Reply:
This is to handle duplicated bus number in gen_pci_enum_policy_info(). Not an issue, but let me remove duplicated number to avoid confusion.